packages/verified-fetch/test/utils/parse-url-string.spec.ts

```diff
@@ -75,8 +75,9 @@ describe('parseUrlString', () => {
           ipns,
           logger
         })
-      ).to.eventually.be.rejected
-        .with.property('message', 'Could not parse PeerId in ipns url "mydomain.com", Non-base64 character')
+      ).to.eventually.be.rejected.and.to.have.nested.property('errors[0]').to.deep.equal(
+        new TypeError('Could not parse PeerId in ipns url "mydomain.com", Non-base64 character')
+      )
```

Comment:
```suggestion
      ).to.eventually.be.rejected
        .with.nested.property('errors[0].message', 'Could not parse PeerId in ipns url "mydomain.com", Non-base64 character')
```

Reply:
What's the error you are seeing? It passes for me..

Reply:
you know what, i read this on my phone. looking from the PC, this is probably fine. will test.

Reply:
idk what happened but reverting it back to the original `.to.eventually.be.rejected.with.property('message', 'Could not parse PeerId in ipns url "mydomain.com", Non-base64 character')` works.